tests/test_utils.py

```diff
@@ -677,6 +688,73 @@ def test_psr_utils():
         0.07774704753236616 * u.G,
     )
 
+    # companion mass
+    assert np.isclose(
+        companion_mass(pb, x, inc=90 * u.deg, mpsr=1.4 * u.Msun),
+        0.2906422269961084 * u.Msun,
+    )
+
+    # pulsar mass
+    assert np.isclose(
+        pulsar_mass(pb, x, 0.4 * u.Msun, inc=90 * u.deg), 2.329629042176839 * u.Msun
+    )
+
+
+@given(
+    floats(min_value=0.5, max_value=3),
+    floats(min_value=0.01, max_value=10),
+    floats(min_value=0.04, max_value=1000),
+    floats(min_value=0.1, max_value=90),
+)
+def test_masses(Mpsr, Mc, Pb, incl):
+    """
+    test pulsar and companion mass calculations for a range of values
+    """
+    assume(incl > 0)
+    assume(incl <= 90)
+    assume(Mpsr > 0)
+    assume(Mc > 0)
+    assume(Pb > 0)
+
+    from pint.utils import (
+        companion_mass,
+        pulsar_mass,
+    )
```

Comment:
Would it be clearer to immediately attach units to Mc, Mpsr, et cetera? Then units could be more or less omitted from the rest of this.

Reply:
I think maybe I didn't make my suggestion clear? My idea was that the first lines of the test would ensure that all the inputs were immediately converted to Quantities, rather than repeatedly converting them on the spot everywhere they are used.

Reply:
I thought you were talking about whether the function (not the test) accepted values that had no units and made assumptions about defaults.  

I am not entirely sure what you mean about the test.  Maybe better to discuss at the next meeting (next week?).

Reply:
My suggestion would be to add the lines, here at the top of the function, 
```python
Mpsr = Mpsr*u.solMass
Mc = Mc*u.solMass
Pb = Pb*u.day
incl = incl*u.deg
```
Then the rest of the function would use Quantities consistently without having to write units mixed into things like `a = (c.G * Mtot * (Pb * u.day / (2 * np.pi)) ** 2) ** (1.0 / 3)` (no units for `Mtot` because it got units when created, but units for `Pb` because it doesn't have them).

Alternatively a bit of functional-programming hocus-pocus could arrange for hypothesis to produce values that are already Quantities, but this is simpler I think.

Reply:
Sure - I don't see a big difference, and if this is clearer it's fine.  Just pushed up those changes.